index.html

```diff
@@ -0,0 +1,200 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" type="text/css" href="./style.css" />
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css" />
+  <title>Joshua Redman - World Tour 2023</title>
+</head>
+
+<body>
+  <header>
+    <div class="bg-dark grid">
+      <ul class="white container f-row js-left g-20 ai-center fs-20">
+        <li>
+          <a href="#"><i class="bi bi-facebook lighter"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="bi bi-twitter lighter"></i></a>
+        </li>
+        <li class="fs-18 underline-hover"><a href="#">Español</a></li>
+      </ul>
+    </div>
+    <nav class="bg-white grid">
+      <div class="container f-row ai-center jc-between" id="navbar__container">
+        <div class="f-row jc-between" id="navbar__button-logo">
+        <div id="hamburger">
+          <i class="bi bi-list fs-40"></i>
+        </div>
+        <a href="#">
+          <img id="navbar__logo" src="./visuals/logo-jrwt.png" />
+        </a>
+      </div>
+          <ul class="f-col g-20 fs-20 fw-bold w-84 m-25" id="navbar__hamburguer__menulist">
+            <li class="btn-grey"><a class="w-100 ta-center" href="#">Home</a></li>
+            <li class="btn-grey"><a class="w-100 ta-center" href="#dates">Dates</a></li>
+            <li class="btn-grey"><a class="w-100 ta-center" href="#band">Band</a></li>
+            <li class="btn-grey"><a class="w-100 ta-center" href="#">Music</a></li>
+            <li class="btn-orange"><a class="w-100 ta-center" href="#">Buy tickets</a></li>
+            <li class="btn-grey"><a class="w-100 ta-center" href="./about.html">About Us</a></li>
+          </ul>
+        <ul class="f-row g-70 jc-around ai-center fs-18 grey" id="navbar__menulist">
+          <li class="orange-hover"><a href="#">Home</a></li>
+          <li class="orange-hover"><a href="#dates">Dates</a></li>
+          <li class="orange-hover"><a href="#band">Band</a></li>
+          <li class="orange-hover"><a href="#">Music</a></li>
+          <li>
+            <button class="btn-orange fs-18 ff-1 ai-center">
+              <a href="#">Buy tickets</a>
+            </button>
+          </li>
+          <li class="orange-hover"><a href="./about.html">About Us</a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <section class="grid" id="front-page">
+      <div class="container f-col g-50" id="front-page__content">
+        <div class="f-col g-20">
+          <div class="f-col g-20 orange" id="front-page__title">
+            <h2 class="ff-3 fs-60">"The Unforgettable Odyssey"</h2>
+            <h1 class="ff-2 fs-100 fw-bold">JOSHUA</h1>
+            <h1 class="ff-2 fs-100 fw-bold w-50 ta-right">REDMAN</h1>
+            <h1 class="ff-2 fs-80 fw-lighter ta-right max-w-66" id="front-page__title__world-tour">
+              WORLD TOUR<br />2023
+            </h1>
+          </div>
```

Comment:
Great design @grauJavier 🤝

- [ ] As stated in the project requirements, we are supposed to stick to the design guidelines as much as possible. Kindly ensure this text section is well-aligned and replicates the design provided.
- Your design
![your home-page](https://github.com/grauJavier/capstone-project/assets/63917725/6bf1f598-5f00-4f08-8026-654293e33eee)
- Given design
![given-home-page](https://github.com/grauJavier/capstone-project/assets/63917725/9cbc8308-d076-479f-9d4a-2b48885f646e)